packages/mcp/src/configuration.ts

```diff
@@ -157,3 +157,17 @@ To fix this you have three different options:
     );
   }
 }
+
+/**
+ * @deprecated MCPConfigurationOptions is deprecated and will be removed in a future release. Use MCPClientOptions instead.
+ */
+export interface MCPConfigurationOptions {
+  id?: string;
+  servers: Record<string, MastraMCPServerDefinition>;
+  timeout?: number; // Optional global timeout
+}
+
+/**
+ * @deprecated MCPConfiguration is deprecated and will be removed in a future release. Use MCPClient instead.
+ */
+export class MCPConfiguration extends MCPClient {}
```

Comment:
Could add something like:
```suggestion
export class MCPConfiguration extends MCPClient {
  constructor(args) {
  	super(args)
  	this.logger.warn(`MCPConfiguration has been renamed to MCPClient and MCPConfiguration is deprecated. The API is identical but the MCPConfiguration export will be removed in the future. Update your imports now to prevent future errors.`)
  }
}
```